docs/conf.py

```diff
@@ -122,5 +123,10 @@ def setup(app):
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme_path = ["_themes"]
-html_theme = "nature_with_gtoc"
+html_theme = "pydata_sphinx_theme"
+
+html_theme_options: {
+    "github_url": "https://github.com/OpenSourceEconomics/respy",
```

Comment:
I think this isn't working correctly? I can't find the GitHub button on the page.

Reply:
Yes, because I used a ``:`` instead of a ``=`` :).